libs/utils/utils.js

```diff
@@ -595,11 +595,24 @@ export async function loadDeferred(area, blocks, config) {
 }
 
 export function loadPrivacy() {
+  const domains = {
+    'adobe.com': '7a5eb705-95ed-4cc4-a11d-0cc5760e93db',
+    'hlx.live': '926b16ce-cc88-4c6a-af45-21749f3167f3',
+    'hlx.page': '3a6a37fe-9e07-4aa9-8640-8f358a623271',
```

Comment:
It's something OneTrust requires - each domain has a configuration, but both hlx.live and hlx.page have the same config. We might be able to prefix with `-test` directly, but this could create issues with custom dns provider.